component/storage/couchdb/store.go

```diff
@@ -191,7 +191,7 @@ func (p *Provider) Close() error {
 // StoreCouchDB represents a CouchDB-backed database.
 type StoreCouchDB struct {
 	log logger
-	db  *kivik.DB
+	DB  *kivik.DB
```

Comment:
Just curious - which linter was complaining about having un-exported field ?

Reply:
It is about tests. There is no way to create an index. See https://github.com/hyperledger/aries-framework-go-ext/pull/13/files#diff-f23f647e13d912c2545a3a269868819dfc85656e02a8ce4072471e9c2e642bc9R219

Reply:
@soluchok the test can instantiate a separate `kivik.DB` instance and create the index with that.

Reply:
Fixed. One test was removed. Not the critical one but affects coverage a bit.